protocol/common/message_sender.go

```diff
@@ -351,6 +356,12 @@ func (s *MessageSender) sendPrivate(
 	messageID := v1protocol.MessageID(&rawMessage.Sender.PublicKey, wrappedMessage)
 	rawMessage.ID = types.EncodeHex(messageID)
 
+	if rawMessage.BeforeDispatch != nil {
```

Comment:
Just wondering if there is something like js optional changing in go?

Reply:
I don't think there's anything like that I am afraid